docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/redshift-loader/index.md

```diff
@@ -13,6 +13,8 @@ import CodeBlock from '@theme/CodeBlock';
 
 Refer to our [guide](/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-aws/setup-destinations/setup-redshift/index.md) on how to setup and launch a Redshift cluster.
 
+`events` table and database schema will be created automatically by loader. Database schema name will be the one in the config. Table name will be `events` and it can't be changed. Keep in mind that Redshift loader user needs to have permission to create schemas on given database to be able to perform this operation. Check [this page](https://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html) for more information about granting privileges in Redshift.
```

Comment:
Same suggestions as in the other one. Actually, you could probably make this a reusable block (you can put it in `.../rdb/loading-transformed-data/_automatic-schema-creation.md`). See `docs/reusable/telemetry/_index.md` and its uses for an example of a reusable block with parameters.